catalog/USGS/USGS_NLCD_RELEASES_2021_REL_TCC_v2021-4.jsonnet

```diff
@@ -0,0 +1,317 @@
+
+    * **Foga, S., Scaramuzza, P.L., Guo, S., Zhu, Z., Dilley, R.D., Beckmann,
+    T., Schmidt, G.L., Dwyer, J.L., Hughes, M.J., Laue, B., 2017.** Cloud
+    detection algorithm comparison and validation for operational Landsat data
+    products. In Remote Sensing of Environment.  *Science Direct*, 194: 379-390
+    [doi:10.1016/j.rse.2017.03.026](http://doi.org/10.1016/j.rse.2017.03.026)
+
+    * **U.S. Geological Survey, 2019.** USGS 3D Elevation Program Digital Elevation 
+    Model, accessed August 2022 at https://developers.google.com/earth-engine/datasets/catalog/USGS_3DEP_10m
+
+    * **Kennedy, R. E., Yang, Z., and Cohen, W. B., 2010.**
+    Detecting trends in forest disturbance and recovery using yearly Landsat time series: 1.
+    LandTrendr - Temporal segmentation algorithms. In Remote Sensing of Environment.
+    *Science Direct*,
+    114(12): 2897-2910
+    [doi:10.1016/j.rse.2010.07.008](https://doi.org/10.1016/j.rse.2010.07.008)
+
+    * **Kennedy, R., Yang, Z., Gorelick, N., Braaten, J., Cavalcante, L., Cohen, W., and Healey, S., 2018.**
+    Implementation of the LandTrendr Algorithm on Google Earth Engine. In Remote Sensing. *MDPI*,
+    10(5): 691
+    [doi:10.3390/rs10050691](https://doi.org/10.3390/rs10050691)
+
+    * **Sentinel-Hub, 2021.**
+    Sentinel 2 Cloud Detector. [Online].
+    Available at: [https://github.com/sentinel-hub/sentinel2-cloud-detector](https://github.com/sentinel-hub/sentinel2-cloud-detector)
+
+    * **Zhu, Z., and Woodcock, C. E., 2012.**. 
+    Object-based cloud and cloud shadow detection in Landsat imagery. In Remote Sensing of
+    Environment. *Science Direct*, 
+    118: 83-94
+    [doi:10.1016/j.rse.2011.10.028](https://doi.org/10.1016/j.rse.2011.10.028)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id) + [
+    ee.link.license('https://data.fs.usda.gov/geodata/rastergateway/treecanopycover/')
+  ],
+  keywords: [
+    'forest',
+    'gtac',
+    'landsat_derived',
+    'redcastle_resources',
+    'sentinel2_derived',
+    'time_series',
+    'usda',
+    'usfs',
+    'usgs',
+  ],
+  providers: [
+    ee.producer_provider('USDA Forest Service (USFS) Geospatial Technology and Applications Center (GTAC)', 
+    'https://www.fs.usda.gov/about-agency/gtac'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(-135.286387, 20.38379, -56.446306, 52.459364,
+                    '2008-06-01T00:00:00Z', '2021-09-30T00:00:00Z'),
+  summaries: {
+    'gee:schema': [
+      {
+        name: 'study_area',
+        description: |||
+          TCC currently covers CONUS, Southeastern Alaska, Puerto Rico-US
+          Virgin Islands and Hawaii. This version contains all study areas.
+          Possible values: 'CONUS, SEAK, PRUSVI, HI'
+        |||,
+        type: ee_const.var_type.string,
+      },
+    ],
+    gsd: [30],
+    'eo:bands': [
+      {
+        name: 'Science_Percent_Tree_Canopy_Cover',
+        description: |||
```

Comment:
please add 
'gee:units': units.percent,
to percentage band

Example: https://github.com/google/earthengine-catalog/blob/main/catalog/USGS/USGS_NLCD.jsonnet#L288

Reply:
Added the gee:units